.github/ISSUE_TEMPLATE/new-feature.yml

```diff
@@ -0,0 +1,55 @@
+name: New Content
+description: For new features or content suggestions
+title: '[NEW]: '
+labels: ['new content']
+body:
+  - type: markdown
+    attributes:
+      value: Thanks for taking the time to submit this issue. Your input is really appreciated.
+  - type: dropdown
+    id: issue-priority
+    attributes:
+      label: Priority
+      description: Choose one of the options
+      options:
+        - Low
+        - Medium
+        - High
+    validations:
+      required: true
+  - type: textarea
+    id: issue-description
+    attributes:
+      label: Description
+      description: Outline what needs to be changed.
+      placeholder:
+    validations:
+      required: true
+  - type: textarea
+    id: issue-impact
+    attributes:
+      label: How will this impact users?
```

Comment:
Not sure if this question is necessary either, as it's often self-explanatory. If it's a simple and common update like documenting a new API method or CLI option, it's simply matching the documentation to the code functionality and doesn't need an impact explanation.